CODE_STYLE.md

```diff
@@ -126,6 +126,25 @@ Use `re_error::format(err)` when displaying an error.
 
 
 ## C++
+For compatibility reasons we use C++11 (and nothing newer).
+We use `clang-format` to enforce most style choices (see [`.clang-format`](.clang-format)).
```

Comment:
recent debacles make me thing we have to specify the exact version :(
At least _somewhere_. Doesn't look like clang-format file can specify a version

Reply:
Maybe as a comment in the clang format file and refer to that?

Reply:
let's wait until pixi saves us from this hell